assets/stylesheets/README.md

```diff
@@ -5,6 +5,8 @@ Calypso uses Sass to build its CSS and compile it to `public/style.css`. Remembe
 
 ### Adding a new Sass file
 
-If you are adding a new Sass file to `assets/stylesheets` you will need to reference the file in `assets/stylesheets/style.scss` for it to load. We have three directories to organize the respective files: layout, sections, and shared. If you are adding a new file you are likely adding it to `sections`.
+If you are adding a new Sass file to `assets/stylesheets` you will need to reference the file in `assets/stylesheets/style.scss` for it to load. We have two directories to organize the respective files: sections and shared. If you are adding a new file you are likely adding it to `sections`.
```

Comment:
Not specifically relevant to this pull request, but adding to `sections` has been strongly discouraged for a while and I think we ought to remove the note.